### ISSUE TITLE
What is the difference between the config for training WavTokenizer-small and WavTokenizer-large?

### ISSUE THREAD

**Issue by handsomelys** [jishengpeng/WavTokenizer#25]
I am curious about what parts of the config need to be modified when WavTokenizer trains the Large version on a larger data set? Could you please give me a reference configuration? In addition, can you give a reference loss value regarding whether the model has converged during training, including the loss of generator and discriminator? I eagerly anticipate your response.

**Comment by jishengpeng** [comment]
> I am curious about what parts of the config need to be modified when WavTokenizer trains the Large version on a larger data set? Could you please give me a reference configuration? In addition, can you give a reference loss value regarding whether the model has converged during training, including the loss of generator and discriminator? I eagerly anticipate your response.

You can directly utilize the configuration of the small version to run the large version, although the increased data volume would typically necessitate a corresponding increase in model parameters. However, the WavTokenizer's parameter configuration already approaches 200M. If you have abundant computational resources, you can attempt to adjust the parameter quantity, and the encoder side can be further expanded. Moreover, it is recommended to evaluate the convergence based on the validation set. In our experiments, the generator's final loss converges to around 38, the discriminator's loss converges to around 10, and the total loss converges to around 25.

**Comment by handsomelys** [comment]
> > I am curious about what parts of the config need to be modified when WavTokenizer trains the Large version on a larger data set? Could you please give me a reference configuration? In addition, can you give a reference loss value regarding whether the model has converged during training, including the loss of generator and discriminator? I eagerly anticipate your response.
> 
> You can directly utilize the configuration of the small version to run the large version, although the increased data volume would typically necessitate a corresponding increase in model parameters. However, the WavTokenizer's parameter configuration already approaches 200M. If you have abundant computational resources, you can attempt to adjust the parameter quantity, and the encoder side can be further expanded. Moreover, it is recommended to evaluate the convergence based on the validation set. In our experiments, the generator's final loss converges to around 38, the discriminator's loss converges to around 10, and the total loss converges to around 25.

Thx!
